scripts/set_assignees.py

```diff
@@ -93,30 +94,103 @@ def process_pr(gh, maintainer_file, number):
             continue
 
         all_areas.update(areas)
-        is_instance = False
+        #
+        # DJL: Don't understand what "is_instance" is trying to record?
+        # It mucks with the weighting though.
+        #
+        #is_instance = False
```

Comment:
@nashif you added this part but it isn't clear to me what you were trying to accomplish with this?

Reply:
IIUC the idea was to give a priority to platforms compared to subsystem. For example if you change the mcux i2c driver, the intent would be to assign it to the mcux maintainer, not the i2c one.

Reply:
so what I've noticed is that we (NXP) will have a PR where most of the code is touching the platform level drivers and such but if you touch one sample then it will bias the assignee to the samples.